client/src/components/ModalVideo.js

```diff
@@ -1,33 +1,108 @@
-import React from "react";
+import React, { useState } from "react";
 import Button from "react-bootstrap/Button";
 import Modal from "react-bootstrap/Modal";
+import Nav from "react-bootstrap/Nav";
+import Tab from "react-bootstrap/Tab";
+import ReactPlayer from "react-player";
+
 import VideoCard from "./VideoCard";
 
-function ModalVideo({ title, url, location, show, handleClose }) {
+function ModalVideo({
+	title,
+	url,
+	audioUrl,
+	description,
+	show,
+	handleClose,
+	location,
+}) {
+	const [activeTab, setActiveTab] = useState("video");
+
+	const handleTabChange = (tab) => {
+		setActiveTab(tab);
+	};
+
+	let modalContent;
 
-  return (
-    <>
+	if (url && description) {
+		modalContent = (
+			<Tab.Container activeKey={activeTab}>
+				<Nav variant="tabs">
+					<Nav.Item>
+						<Nav.Link eventKey="video" onClick={() => handleTabChange("video")}>
+							Video
+						</Nav.Link>
+					</Nav.Item>
+					<Nav.Item>
+						<Nav.Link
+							eventKey="description"
+							onClick={() => handleTabChange("description")}
+						>
+							Description
+						</Nav.Link>
+					</Nav.Item>
+				</Nav>
+				<Tab.Content>
+					<Tab.Pane eventKey="video">
+						<VideoCard url={url} />
+					</Tab.Pane>
+					<Tab.Pane eventKey="description">{description}</Tab.Pane>
+				</Tab.Content>
+			</Tab.Container>
+		);
+	} else if (audioUrl && description) {
```

Comment:
For example, if you remove this `else if` and let things fall to the `else` statement, could you produce the same result?

Reply:
For speed I'll be explicit: if you delete this `else if` and swap `{description && <div>Description: {description}</div>}` to below the audioUrl statement that ends line 74, you'll have the same thing with simpler logic.

At that point I'd be ready to approve.